### DISCUSSION TITLE
AAD Authentication Plug-in suddenly no longer installed

### DISCUSSION THREAD

**Discussion by astro-phage** [hkamel/sonar-auth-aad#147] [Q&A]
Hi

We are using a self-hosted (Azure Cloud) instance of "_SonarQube Community EditionVersion 9.4 (build 54424)_".  

We had installed "_Azure Active Directory (AAD) Authentication Plug-in for SonarQube_" via Administration / Marketplace about 1 month ago and configured everything to work with AAD.

Suddenly, the option to log in via AAD is no longer available and when I check the SonarQube instance Marketplace, the plugin is no longer installed. I am the only one who has access to it, and I haven't changed anything in a long time. 

Does anyone have any idea why this happened and if it could happen again? The only explanation I can remotely think of is that a new version on the marketplace uninstalls the current version?

Thanks a lot in advance!

**Comment by srvrguy** [comment]
A new release was made, and that might have made it disappear from the marketplace for older versions. You _should_ be able to see it in the marketplace section if you change the filter to "installed". At least, I have some plugins installed on my test system that aren't available via the marketplace and they show there.

At the very least, if the settings for the plugin still show, it's still installed. SonarQube doesn't look at the marketplace listing when loading things, just the jars in the plugin directory.
